vhdbuilder/packer/components.json

```diff
@@ -136,7 +136,7 @@
       "amd64OnlyVersions": [],
       "multiArchVersions": [
         "ciprod05192022",
-        "ciprod06142022"
+        "ciprod06272022"
```

Comment:
I think should replace "ciprod05192022" with the new image "ciprod06272022"?
not the "ciprod06142022", I suppose "ciprod06142022" is currently in use

Reply:
> I think should replace "ciprod05192022" with the new image "ciprod06272022"?
> not the "ciprod06142022", I suppose "ciprod06142022" is currently in use

>  I haven’t released ciprod06142022 hence I have updated this one with ciprod06272022.